lib/octicons_helper/README.md

```diff
@@ -10,12 +10,12 @@ This rails helper lets you easily include svg octicons in your rails apps.
 1. Add this to your `Gemfile`
 
     ```rb
-    gem 'octicons_helper'
+    gem 'openproject-octicons_helper'
     ```
 
 3. Use this tag in your erbs
 
     ```rb
     <%= octicon "alert", :height => 32,  :class => "right left", :"aria-label" => "hi" %>
     ```
-We recommend including the CSS in the [`@primer/octicons`](/packages/javascript) npm module. You can also npm install that package and include `build/build.css` in your styles.
\ No newline at end of file
+We recommend including the CSS in the [`@openproject/octicons-angular`](/packages/javascript) npm module. You can also npm install that package and include `build/build.css` in your styles.
```

Comment:
The mentioned file `build/build.css` does not exist in our package. Is that intended?